EosPayload/drivers/radio_driver.py

```diff
@@ -94,6 +94,9 @@ def xbee_send_callback(_client, _userdata, message):
                 new_transmit_header = TransmitHeader(self.sequence_number)
                 packet_from_mqtt.transmit_header = new_transmit_header
 
+                # Store to data file
+                self.data_log(["sending",packet_from_mqtt.encode_to_string()])
```

Comment:
1. should also log receives if possible
2. welp, we yeeted encode_to_string starting in 4.0.  Maybe we need to ticket adding it back.  Maybe

Reply:
1. Receives are already recorded via logger, and data_log is apparently not thread safe.
2. Yeah, I think there's a legitimate use case for turning packets into strings, less so strings into packets, so it might be better to just override __repr__ or something

Reply:
1. Logger is for recording events, not data.  Esp. since the logger is so noisy, I'd argue for doing data_log on the receives.  It would be pretty trivial to make it thread safe, just slap a threading.Lock on there, all of 3 lines of code